src/nodes/Table/Table.js

```diff
@@ -14,6 +14,8 @@ import {
 	selectedRect,
 	selectionCell,
 } from '@tiptap/pm/tables'
+// eslint-disable-next-line n/no-extraneous-import
```

Comment:
From my understanding we should try to avoid extraneous imports. Better make the package a direct dependency in `package.json`.

But in this case you should just be able to import `Fragment` from `@tiptap/pm/model`. Tiptap recently introduced the "meta-package" `@tiptap/pm`, which exports all the prosemirror packages at `@tiptap/pm/<name>`.